openfl-tutorials/interactive_api_tutorials_experimental/director_kvasir/director_folder/start_director.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Start director module."""
+
+
+import asyncio
+import logging
+
+from openfl.component.director.director import serve
+from openfl.interface.cli import setup_logging
+
+setup_logging()
+
+logger = logging.getLogger(__name__)
+
+if __name__ == '__main__':
+    sample_shape = ['529', '622', '3']
+    target_shape = ['529', '622']
+    asyncio.run(serve(sample_shape=list(sample_shape), target_shape=list(target_shape),
```

Comment:
The details of `asyncio.run(serve(...)))` should be hidden behind a higher level OpenFL abstraction. Calling a start method on a director object would be a natural way to use this. i.e.

```
director = Director(sample_shape=sample_shape, 
         target_shape=target_shape,
         disable_tls=False, 
         root_ca='./cert/root_ca.crt', 
         key='./cert/localhost.key', 
         cert='./cert/localhost.crt', 
         ))
director.start()
```

Reply:
^ This will bring the initialization of the director server inline with the envoy in the provided examples

Reply:
Changed